db/migrate/20160415121724_create_cloud_services.rb

```diff
@@ -0,0 +1,21 @@
+class CreateCloudServices < ActiveRecord::Migration[5.0]
+  def change
+    create_table :cloud_services do |t|
+      t.string :ems_ref
+      t.string :source
+      t.string :binary
+      t.string :host_name
+      t.string :status
+      t.string :scheduling_status
+      t.string :zone_name
```

Comment:
Is this a zone from the zones table, or is this something specific to cloud services, or is this an availability_zone?  There is some ambiguity with the naming, and I'm hoping a clarification might evoke the right column name.